public/javascript/pump/socket.js

```diff
@@ -26,6 +26,13 @@
 
     "use strict";
 
+    // Private instance
```

Comment:
I assume you mean this variable is private to this module? If so, no need to explicitly mention it - it's clear enough from the closure wrapper and the initial underscore.

Reply:
Only to be clear and difference the `sock` instance

Reply:
Okay. I'd just get rid of this comment then.

Reply:
@vxcamiloxv this comment still needs to go away :)

Reply:
it'll leaves us